### PR TITLE
README: add Addon Manager install method + clarifications

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,8 +30,17 @@ FreeCAD > v0.16
...
+### Addon Manger  
+Starting from v0.17 it's possible to use the built-in FreeCAD [Addon Manager](https://github.com/FreeCAD/FreeCAD-addons#1-builtin-addon-manager)
+located in the `Tools` > `Addon Manager` dropdown menu.
+
+### pip
+
 `pip install https://github.com/looooo/FCGear/archive/master.tar.gz`
 
+**Important note:** Make sure the `pip` you're using is used by FreeCAD as well. It's recomemnded to use it
+separately from your system's python/pip.
```
Comment (looooo) [inline README.md]: hmm, not sure if this recommendation is really needed here. For example if we have installed freecad via ppa have to use the system python/pip.

Comment (luzpaz) [reply README.md]: Then perhaps change the note to reflect the importance of using the same python/pip version that FC was installed with ?

Comment (looooo) [reply README.md]: see https://github.com/looooo/freecad.gears/pull/30/commits/9654d8a97c9617954ad7b3a5fe52a66b99534a6b. If you are happy with this, I will merge.

Comment (luzpaz) [reply README.md]: fantastic :smile:

**Pull request merged**